### PR TITLE
[DSI-7258] Support Console to Match Entra Invitation Email Template

### PR DESCRIPTION
**Pull request by Lakorthus**
## Summary
Jira ticket: [DSI-7258](https://dfe-secureaccess.atlassian.net/browse/DSI-7258)

## Changes
- Added Entra to schema and local config
- Added Entra flag to Support console to match invitation email template
- Removed unused parameters

## Related PRs:
- https://github.com/DFE-Digital/login.dfe.dsi-config/pull/35

### CODE REVIEW

File: src/app/users/views/confirmNewUser.ejs [hunk 1]
```diff
@@ -75,21 +75,34 @@
...
-
-                            <p>To verify your email address and set up your account, please follow the link below: <em>LINK TO THE SERVICE </em></p>
-
-                            <p><em>If you received this email in error, please ignore it.</em></p>
-
-                            <p>Kind regards,</p>
-
-                            <p>DfE Sign-in</p>
-                        </div>
+                        <% if (locals.entra && locals.entra.enableEntraSignIn === true) { %>
```
Comment (ipredoi) [inline src/app/users/views/confirmNewUser.ejs]: Can we change this line to match the flag for user registration 
`<% if (locals.entra && locals.entra.useEntraForAccountRegistration === true){ %>`

File: src/index.js [hunk 2]
```diff
@@ -107,6 +107,9 @@ const init = async () => {
       interactions: config.hostingEnvironment.interactionsUrl,
       assets: assetsUrl,
     },
+    entra: {
+      enableEntraSignIn: config.entra.enableEntraSignIn,
```
Comment (ipredoi) [inline src/index.js]: Can we change this to 
`useEntraForAccountRegistration: config.entra.useEntraForAccountRegistration`

File: src/infrastructure/config/schema.js [hunk 3]
```diff
@@ -119,7 +119,15 @@ const togglesSchema = new SimpleSchema({
 });
 
 const notificationsSchema = new SimpleSchema({
-  connectionString: patterns.redis
+  connectionString: patterns.redis,
+});
+
+const entraSchema = new SimpleSchema({
+  enableEntraSignIn: {
```
Comment (ipredoi) [inline src/infrastructure/config/schema.js]: Can we change it to `useEntraForAccountRegistration`

**Review by ipredoi — APPROVED** [review]
looks good

**Review by andreaspetridis11 — APPROVED** [review]

**Pull request merged**